.github/workflows/build.yml

```diff
@@ -424,7 +435,7 @@ jobs:
         name: Install Go
         uses: actions/setup-go@v3
         with:
-          go-version: ${{ matrix.go-version }}
+          go-version: 1.22.x
```

Comment:
Should update the matrix version instead.

Reply:
The matrix does not exist for this job. Our action linter caught this.